### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by Kaiserabbas**
Hi Good Day, Welcome to my pull request,
- Removed all hardcoded items from the tasks array.
- Created a new JavaScript file for the new functionality.
- Implemented a function for adding a new task (add a new element to the array).
- Implemented a function for deleting a task (remove an element from the array).
- Implemented a function for editing task descriptions.
- By default new tasks property index is set to the value of the new array length.
- Deleting a task updates all remaining items' indexes. 
- All changes to the To Do List are saved in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
-/* eslint-disable no-unused-vars */
+/* eslint-disable */ /*importing files from the modules*/
 import _ from 'lodash';
```
Comment (omar-muhamad) [inline src/index.js:2]: - [ ] Please remove any eslint disablement lines inside your code. Linters is a great tool that helps you write a clean and more readable code so disabling it is not a good practice.

File: src/index.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 import './index.css';
-import populateTodoList from './print.js';
+import { saveTasks, addTasks, showTasks, tasksForm } from './print.js';
 
 document.addEventListener('DOMContentLoaded', () => {
-  populateTodoList();
+  saveTasks();
+  showTasks();
+  tasksForm.addEventListener('submit', addTasks);
+  tasksForm.addEventListener('submit', showTasks);
```
Comment (omar-muhamad) [inline src/index.js:10]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Project%20requirements-,Add%20a%20new%20JavaScript%20file%20and%20import%20it%20as%20a%20module,-%3A), Please add methods related to the status updates (completed: true / false) in a new module.

- [ ] Kindly add an event listener to the checkbox (change) that updates the items object's value for completed key upon user actions.

- [ ] Please Implement a function for the "Clear all completed" button (use filter() method).

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Kaiserabbas,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @Kaiserabbas 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️

-  [ ]  It will be better to implement correctly the functionality of the trash icon, which is not working properly when you click on the checkbox to delete the item that must appear so that you can delete each icon independently. please fix that to make your  app  easy for using :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Kaiserabbas** [comment]
trash icon is functional,  when you click on the checkbox to delete the item it appears. you can delete each icon independently.

**Review by christianbiring1 — APPROVED** [review]
Hi @Kaiserabbas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] Great work on implementing the feature to add and delete tasks :+1: 

## Optional suggestions
- [ ] Kindly make sure your trash icon is visible, you may consider readjusting your colors
- [ ] Also make sure the process to operate or interact with the app is quite similar with the one in the project description. or you may consider just having the trash icon present even without checking the box 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**